main/preload.js

```diff
@@ -22,14 +23,19 @@ contextBridge.exposeInMainWorld('electron', {
     isReady: () => ipcRenderer.invoke('saturn:isReady'),
     getLog: () => ipcRenderer.invoke('saturn:getLog'),
     getWebUrl: () => ipcRenderer.invoke('saturn:getWebUrl'),
-    getFilAddress: () => ipcRenderer.invoke('saturn:getFilAddress'),
-    setFilAddress: (/** @type {string | undefined} */ address) => ipcRenderer.invoke('saturn:setFilAddress', address)
+    getFilAddress: () => ipcRenderer.invoke('saturn:getFilAddress'), // soon to be removed
```

Comment:
why are these "soon to be removed"? Since this PR is implementing the updated wallet backend interface, can we just remove the deprecated calls?

Reply:
Mainly because if it's dependency on `saturn-node.js` instead of `station-config.js`.
In order to completely replace this `getFilAddress` by `getStationWalletAddress`, `saturn-node.js` needs a refactor first